10up/publisher-media-kit
# Replace hardcoded image URLs with PUBLISHER_MEDIA_KIT_URL.

### Description of the Change


Updates hard coded image URLs in page template to use the `PUBLISHER_MEDIA_KIT_URL` constant defined in the main plugin file. This ensures that the images of the generated pages are correctly referenced no matter the file structure of the site. This allows for:

* WordPress installed in sub-directories such as example.com/blog
* WordPress installs using a custom content directory such as `/content`.


Closes #111

### How to test the Change


1. Install WordPress as you normally would
2. Rename the `wp-content` directory to `content`
3. Add the following to your `wp-config.php` file (replacing the values if they already exist):
   ```php
   // ========================
   // Custom Content Directory
   // ========================
   define( 'WP_CONTENT_DIR', dirname( __FILE__ ) . '/content' );
   define( 'WP_CONTENT_URL', 'http://' . $_SERVER['HTTP_HOST'] . '/content' );
   ```
4. If you already have a media kit page created by PMK, trash it.
5. Deactivate and reactivate the plugin
6. On `develop` the images will be broken, on this branch the images will display as expected.


### Changelog Entry

> Fixed - Image display on WordPress installs with non-standard content paths.


### Credits

Props @peterwilsoncc, @dkotter, @adriantry


### Checklist:
